src/treestatus/api/treestatus_api/api.py

```diff
@@ -36,7 +51,184 @@ def _now():
     return datetime.datetime.utcnow().replace(tzinfo=pytz.UTC)
 
 
+def _statuspage_data(
+    close,
+    component_id,
+    tree,
+    status_from,
+    status_to,
+):
+    data = {
+        'status': close and 'resolved' or 'investigating',
+        'components': {
+            component_id: close and 'operational' or 'major_outage',
+        }
+    }
+    if not close:
+        data['name'] = f'Tree {tree.tree} closed'
```

Comment:
Especially it reads weird here, when `close` is `False`, but the message mentions "closed".